main.go

```diff
@@ -197,6 +197,7 @@ var (
 	}
 	slowLinters = []string{"structcheck", "varcheck", "errcheck", "aligncheck", "testify", "test", "interfacer", "unconvert", "deadcode"}
 	sortKeys    = []string{"none", "path", "line", "column", "severity", "message", "linter"}
+	outputKeys  = []string{"console", "json", "checkstyle"}
```

Comment:
Done. I suppose you don't mind I used flag `--checkstyle` instead of `--check-style`. Checkstyle project is usually referred to as a single word rather than a two word term.